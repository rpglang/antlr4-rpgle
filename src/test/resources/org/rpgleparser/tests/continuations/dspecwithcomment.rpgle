  |  D Item_Not_Passed_For_Deletion...
  |  D                 s                   like(*in)