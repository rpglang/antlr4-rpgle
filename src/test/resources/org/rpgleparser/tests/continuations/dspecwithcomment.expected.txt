/*===TOKENS===*/
HIDDEN:LEAD_WS5_Comments <  |  >
DS_FIXED                 <D>
CONTINUATION_NAME        <Item_Not_Passed_For_Deletion>
HIDDEN:CE_LEAD_WS5_Comments<  |  >
NAME                     <>
EXTERNAL_DESCRIPTION     < >
DATA_STRUCTURE_TYPE      < >
DEF_TYPE_S               <s >
FROM_POSITION            <       >
TO_POSITION              <       >
DATA_TYPE                < >
DECIMAL_POSITIONS        <  >
RESERVED                 < >
KEYWORD_LIKE             <like>
'('                      <(>
SPLAT_IN                 <*in>
')'                      <)>
/*===TREE===*/
(r
  (statement
    (dspec
      D
      (ds_name Item_Not_Passed_For_Deletion )
      ' '
      ' '
      's '
      '       '
      '       '
      ' '
      '  '
      ' '
      (keyword
        (keyword_like
          like
          (
          (simpleExpression (identifier (all (symbolicConstants *in))))
          )
        )
      )
      <EOF>
    )
  )
)
/*======*/